webpack.config.js

```diff
@@ -6,11 +7,11 @@ module.exports = {
   entry: {
     index: "./src/index.js",
   },
-  output: {
-    filename: "main.js",
-    path: path.resolve(__dirname, "dist"),
-    clean: true,
-  },
+  // output: {
+  //   filename: "main.js",
+  //   path: path.resolve(__dirname, "dist"),
+  //   clean: true,
+  // },
```

Comment:
- [ ] kindly remove any commented code in your project to keep the code clean and professional 🙏